content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown

```diff
@@ -0,0 +1,126 @@
++++
+title="Amazon ECS"
+type="docs"
+description="How to get started with Bottlerocket on Amazon ECS"
++++
+
+## Introduction
+
+In this quickstart, we use a number of techniques and tools like `jq` and environment variables to make the experience as simple and straightforward as possible.
```

Comment:
too many 'ands': 'techniques and tools' -> 'techniques/tools' 
more specific: 'the experience' -> ' the quickstart experience'